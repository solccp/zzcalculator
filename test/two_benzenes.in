 12
14
C        0.0000000362204327      -0.0000000000000015       1.4000690578049293
C 1.2124953658309454 5.00001 -0.7000345581539054
C -0.0000000359812804 5.00001 -1.4000690560049081
C        1.2124954040702669       0.0000000000000059       0.7000345044112616
C        1.2124953658309454       0.0000000000000054      -0.7000345581539054
C 1.2124954040702669 5.00001 0.7000345044112616
C -1.2124954033844171 5.00001 -0.7000345072502421
C       -0.0000000359812804      -0.0000000000000248      -1.4000690560049081
C       -1.2124954033844171       0.0000000000000273      -0.7000345072502421
C       -1.2124953665278562      -0.0000000000000123       0.7000345588165076
C 0.0000000362204327 5.00001 1.4000690578049293
C -1.2124953665278562 5.00001 0.7000345588165076
