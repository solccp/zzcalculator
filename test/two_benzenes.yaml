---
structure_id: 1
number_of_atoms: 12
double_bonds:
- [1, 10]
- [9, 8]
- [4, 5]
- [7, 12]
- [11, 6]
- [2, 3]
---
structure_id: 2
number_of_atoms: 12
double_bonds:
- [1, 10]
- [9, 8]
- [4, 5]
- [7, 3]
- [2, 6]
- [11, 12]
---
structure_id: 3
number_of_atoms: 12
double_bonds:
- [1, 10]
- [9, 8]
- [4, 5]
rings:
- [7, 3, 2, 6, 11, 12]
---
structure_id: 4
number_of_atoms: 12
double_bonds:
- [1, 4]
- [10, 9]
- [8, 5]
- [7, 12]
- [11, 6]
- [2, 3]
---
structure_id: 5
number_of_atoms: 12
double_bonds:
- [1, 4]
- [10, 9]
- [8, 5]
- [7, 3]
- [2, 6]
- [11, 12]
---
structure_id: 6
number_of_atoms: 12
double_bonds:
- [1, 4]
- [10, 9]
- [8, 5]
rings:
- [7, 3, 2, 6, 11, 12]
---
structure_id: 7
number_of_atoms: 12
double_bonds:
- [7, 12]
- [11, 6]
- [2, 3]
rings:
- [1, 4, 5, 8, 9, 10]
---
structure_id: 8
number_of_atoms: 12
double_bonds:
- [7, 3]
- [2, 6]
- [11, 12]
rings:
- [1, 4, 5, 8, 9, 10]
---
structure_id: 9
number_of_atoms: 12
rings:
- [1, 4, 5, 8, 9, 10]
- [7, 3, 2, 6, 11, 12]
