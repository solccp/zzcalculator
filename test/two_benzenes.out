 4 + 4 x + 1 x^2
 total: 9
